### ISSUE TITLE
Broken/non-existent links on World homepage

### ISSUE THREAD

**Issue by VaugeIntentionsGaming** [worldanvil/worldanvil-bug-tracker#929]
**World Anvil Username**: VagueIntentionsGaming

**World / Campaign / RPG Character / Manuscript Title**: Athil Vestra

**Feature**: direct links to articles

**Related URLS** (To public & visible pages add screenshots otherwise): https://www.worldanvil.com/w/athil-vestra-vagueintentionsgaming

**Describe the Issue**
From the world homepage, neither my followers or myself can access any links, including : World Codex, Recent articles, Timelines, Campaigns or Characters. The only links that function are under Maps.

**To Reproduce**
1. Go to 'World page'
2. Click on 'any link to article'
3. See error - no message, but links don't work

**Expected behavior**
As what worked as of last week, selecting links and folders provided the result of opening the folder on screen or directing the user to the article. As well, selecting the option (as author) to edit the link was functioning. 

**Screenshots**
![Capture](https://user-images.githubusercontent.com/101755156/158699592-83a6d942-3af2-4a79-90ab-4f0e221f22cb.PNG)

**Comment by iamromeo** [comment]
From what I can see this is caused due to broken BBcode, probably on the world introduction - the chances are you have unclosed/mistyped tags, and since everything is centered, it might be a unclosed "[center]" tag. 

But this is just a guess, I cannot really tell without seeing the actual content of the world description. Can you paste it here?

**Comment by VaugeIntentionsGaming** [comment]
I took what you mentioned and reviewed the last article that I created. There was a couple of instances where I had written "[[/b]br]" instead of [/b][br]. I fixed those. 

I reviewed the World description article and it is fairly light on text as well as BBcodes. 

See below:
[in]The lands of Athil Vestra hold might, magic and many interesting peoples and regions for discovery. Formed by an ancient Athalean named Balrond, the planet has experienced much growth and destruction over the ages; from the rise and fall of the first humans, the era of arcane discovery and expansion of the Senovi, to the rise of the dragons and the near extinction of the Senovi under the dragon God Cerixion, to the growth and expansion of the divine pantheon, and to what it has become in recent years.[br]

I have thoroughly reviewed all of my articles and don't appear to notice any similar issues. Checking the world page and the trouble with links has not changed.

Thank you.

[in]With the aid of powerful magic, most of the lands have experienced much technological and cultural advancements. While not always peaceful times, there has been much expansion and advancement of the various races and empires over the millennia.
